File: 1980. Find Unique Binary String/Find Unique Binary String.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        result = ""
        for i in range(len(nums)):
            if nums[i][i] == "0":
                result = result + "1"
            else:
                result = result + "0"
        return result

File: 1980. Find Unique Binary String/Find Unique Binary String.cpp
class Solution {
public:
    string findDifferentBinaryString(vector<string>& nums) {
        string result = "";
        for (int i = 0; i < nums.size(); i++){
            if (nums[i][i] == '0'){
                result = result + "1";
            }
            else{
                result = result + "0";
            }
        }
        return result;
    }
};
